types/create-react-class/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "dt-header": false,
```

Comment:
It looks like there _is_ a DT header in the file; what's the issue here?

Reply:
The issue is that there are no github links in the "Definitions by" field (same with the other package). I can add my own profile URL and re-enable this rule.

Reply:
Actually, this is an issue with putting _any_ non-GitHub URL in the "Definitions by" section (i.e., Microsoft, Asana, etc.). So I could re-enable this, but it would mean having only my name in that section. Any preference?

Reply:
@Microsoft and @Asana are at least on github, so you can replace their corporate urls (which are, IMO, useless in this context) with the github org links.

Reply:
Nvm, as per comment on other package, leave them as-is w/lint rule. Should update the rule to not fail on those domains.

Reply:
OK. Leaving as-is.